[TRTLLM-6785][feat] BREAKING CHANGE Enable TRTLLM sampler by default

dcampora opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Introduced a new option to enable the Torch sampler for model inference, replacing the previous TensorRT LLM sampler flag across user interfaces, configuration files, and command-line arguments.

* **Refactor**
  * Renamed parameters and configuration options from `enable_trtllm_sampler` to `use_torch_sampler` throughout the application and documentation for improved clarity.
  * Updated terminology from "max batch size" to "max number of sequences" in APIs and user-facing documentation for model decoding.
  * Improved internal naming and added optionality to sampler state attributes for better clarity and robustness.
  * Enhanced sampler setup to handle additional request types and improved event handling robustness.

* **Bug Fixes**
  * Enhanced error messaging for out-of-range sampling parameters.
  * Improved robustness against missing configuration attributes and event handling.

* **Tests**
  * Updated all relevant tests and configurations to use the new sampler flag and naming conventions.
  * Adjusted test expectations and fixtures to reflect updated model and configuration options.

* **Documentation**
  * Updated descriptions and statuses in help texts and configuration files to reflect the new Torch sampler option and its maturity level.
  * Simplified example usage by removing deprecated sampler flags.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

<!--
Please write the PR title by following this template:

[JIRA ticket/NVBugs ID/GitHub issue][fix/feat/doc/infra/...] \<summary of this PR\>

For example, assume I have a PR to support a new feature about cache manager for JIRA ticket TRTLLM-1000, it would be like:

[TRTLLM-1000][feat] Support a new feature about cache manager

Or I have a PR to fix a Llama3 accuracy issue:

[https://nvbugs/1234567][fix] Fix Llama3 accuracy issue
-->

## Description

<!--
Please explain the issue and the solution in short.
-->

## Test Coverage

<!--
Please list clearly what are the relevant test(s) that can safeguard the changes in the PR. This helps us to ensure we have sufficient test coverage for the PR.
-->

## GitHub Bot Help

`/bot [-h] ['run', 'kill', 'skip', 'reuse-pipeline'] ...`

Provide a user friendly way for developers to interact with a Jenkins server.

Run `/bot [-h|--help]` to print this help message.

See details below for each supported subcommand.

<details>

`run  [--reuse-test (optional)pipeline-id --disable-fail-fast --skip-test --stage-list "A10-PyTorch-1, xxx" --gpu-type "A30, H100_PCIe" --test-backend "pytorch, cpp" --add-multi-gpu-test --only-multi-gpu-test --disable-multi-gpu-test --post-merge --extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx" --detailed-log --debug(experimental)]`

Launch build/test pipelines. All previously running jobs will be killed.

`--reuse-test (optional)pipeline-id ` *(OPTIONAL)* : Allow the new pipeline to reuse build artifacts and skip successful test stages from a specified pipeline or the last pipeline if no pipeline-id is indicated. If the Git commit ID has changed, this option will be always ignored. The DEFAULT behavior of the bot is to reuse build artifacts and successful test results from the last pipeline.

`--disable-reuse-test ` *(OPTIONAL)* : Explicitly prevent the pipeline from reusing build artifacts and skipping successful test stages from a previous pipeline. Ensure that all builds and tests are run regardless of previous successes.

`--disable-fail-fast ` *(OPTIONAL)* : Disable fail fast on build/tests/infra failures.

`--skip-test ` *(OPTIONAL)* : Skip all test stages, but still run build stages, package stages and sanity check stages. Note: Does **NOT** update GitHub check status.

`--stage-list "A10-PyTorch-1, xxx"` *(OPTIONAL)* : Only run the specified test stages. Examples: "A10-PyTorch-1, xxx". Note: Does **NOT** update GitHub check status.

`--gpu-type "A30, H100_PCIe"` *(OPTIONAL)* : Only run the test stages on the specified GPU types. Examples: "A30, H100_PCIe". Note: Does **NOT** update GitHub check status.

`--test-backend "pytorch, cpp"` *(OPTIONAL)* : Skip test stages which don't match the specified backends. Only support [pytorch, cpp, tensorrt, triton]. Examples: "pytorch, cpp" (does not run test stages with tensorrt or triton backend). Note: Does **NOT** update GitHub pipeline status.

`--only-multi-gpu-test ` *(OPTIONAL)* : Only run the multi-GPU tests. Note: Does **NOT** update GitHub check status.

`--disable-multi-gpu-test ` *(OPTIONAL)* : Disable the multi-GPU tests. Note: Does **NOT** update GitHub check status.

`--add-multi-gpu-test ` *(OPTIONAL)* : Force run the multi-GPU tests in addition to running L0 pre-merge pipeline.

`--post-merge ` *(OPTIONAL)* : Run the L0 post-merge pipeline instead of the ordinary L0 pre-merge pipeline.

`--extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx"` *(OPTIONAL)* : Run the ordinary L0 pre-merge pipeline and specified test stages. Examples: --extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx".

`--detailed-log ` *(OPTIONAL)* : Enable flushing out all logs to the Jenkins console. This will significantly increase the log volume and may slow down the job.

`--debug ` *(OPTIONAL)* : **Experimental feature**. Enable access to the CI container for debugging purpose. Note: Specify exactly one stage in the `stage-list` parameter to access the appropriate container environment. Note: Does **NOT** update GitHub check status.

For guidance on mapping tests to stage names, see `docs/source/reference/ci-overview.md`
and the `scripts/test_to_stage_mapping.py` helper.

### kill

`kill  `

Kill all running builds associated with pull request.

### skip

`skip --comment COMMENT `

Skip testing for latest commit on pull request. `--comment "Reason for skipping build/test"` is required. IMPORTANT NOTE: This is dangerous since lack of user care and validation can cause top of tree to break.

### reuse-pipeline

`reuse-pipeline `

Reuse a previous pipeline to validate current commit. This action will also kill all currently running builds associated with the pull request. IMPORTANT NOTE: This is dangerous since lack of user care and validation can cause top of tree to break.

</details>

Funatiq committed:
refactor: Rename maxBatchSize to maxNumSequences in decoder and gptDecoder classes

- Updated the parameter names and related comments in the DecoderState and GptDecoder classes to reflect the change from maxBatchSize to maxNumSequences.
- Adjustments were made in the setup methods, member variables, and associated bindings in the Python interface.
- This change improves clarity regarding the number of sequences being processed.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>

dcampora committed:
Make TRTLLM Sampler the default.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora committed:
Fix default.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

tensorrt-cicd commented:
[PR_Github #12446](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12446/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #9255](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/9255/) completed with status: 'FAILURE'

dcampora commented on tests/unittest/_torch/test_return_logits.py:
```diff
@@ -83,18 +83,18 @@ def test_generate_with_return_logits(disable_overlap_scheduler: bool,
         pytest.skip("Nothing to test")
 
-    if not enable_trtllm_sampler and gather_context_logits:
+    if not enable_torch_sampler and gather_context_logits:
         pytest.skip("TorchSampler does not support gather_context_logits")
```
Done.

dcampora committed:
Fix test_return_logits.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora committed:
Wider support of executor_config.speculative_config.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

tensorrt-cicd commented:
[PR_Github #12523](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12523/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #9307](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/9307/) completed with status: 'FAILURE'

dcampora commented:
/bot run --disable-fail-fast

dcampora committed:
Set max batch size to 64 in test_llm_api_pytorch qwen
3 test.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora committed:
Fix run of tests against spec decoding)

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #12567](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12567/) [ run ] triggered by Bot

dcampora committed:
Fix max in max_attn_window values.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora committed:
Formatting.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #12679](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12679/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #12676](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12676/) [ run ] completed with state `ABORTED`

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #12679](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12679/) [ run ] completed with state `ABORTED`

dcampora committed:
Remove enable_trtllm_sampler option.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #12713](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12713/) [ run ] triggered by Bot

laikhtewari commented on tests/unittest/_torch/speculative/test_eagle3.py:
```diff
@@ -59,6 +59,7 @@ def test_llama_eagle3(use_cuda_graph: bool, attn_backend: str,
         # that the draft model won't go above its max in warmup
         # in this test.
         max_seq_len=8192,
+        enable_torch_sampler=True,
```
Yes, trtllm sampler as default seems reasonable. sampling params is must have for most whereas some may choose to not use spec dec

dcampora committed:
Unwaive qwen test, adapt sampler output.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run

dcampora committed:
Adapt e2e test format.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #12879](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12879/) [ run ] completed with state `FAILURE`

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #12893](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12893/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #12893](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/12893/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #9611](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/9611/) completed with status: 'FAILURE'

dcampora commented:
/bot run --only-multi-gpu-test

tensorrt-cicd commented:
[PR_Github #13182](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13182/) [ run ] triggered by Bot

dcampora committed:
Use Qwen3/Qwen3-0.6B-Base instead of TinyLlama for its longer max_position_embeddings.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #13193](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13193/) [ run ] triggered by Bot

netanel-haber commented on tensorrt_llm/_torch/pyexecutor/_util.py:
```diff
@@ -586,15 +586,14 @@ def instantiate_sampler(engine: PyTorchModelEngine,
-        return TRTLLMSampler(executor_config, engine.model, engine.dtype,
-                             mapping, decoding_mode,
-                             pytorch_backend_config.disable_overlap_scheduler)
+    if pytorch_backend_config.enable_torch_sampler or pytorch_backend_config.enable_mixed_sampler or engine.spec_config is not None:
```
Can `engine.spec_config is not None` be True while `engine.spec_config.spec_dec_mode.has_spec_decoder(
    )` is False? I think in the previous `if` on L586, `engine.spec_config is not None` is just a type guard, equivalent to optional chaining. Or am I missing something?

netanel-haber commented on tests/unittest/_torch/speculative/test_draft_target.py:
```diff
@@ -41,11 +41,13 @@ def test_llama_draft_target(use_cuda_graph: bool, attn_backend: str):
         max_batch_size=max_batch_size,
         kv_cache_config=kv_cache_config,
         max_num_tokens=2048,
+        enable_torch_sampler=True,
```
Shouldn't these default to TRTLLMSampler by dint of going down the "has_spec_dec" path?

dcampora replied:
If you look at the invocations in this test, it performs one with just `llm_common_config` and one with `LLM(**llm_common_config, speculative_config=spec_config)`. The former will be invoked with TRTLLM Sampler unless explicitly set to Torch sampler.

netanel-haber replied:
My bad, sorry

dcampora committed:
Remove unnecessary enable_torch_sampler in test.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented on tests/unittest/_torch/speculative/test_draft_target.py:
```diff
@@ -41,11 +41,13 @@ def test_llama_draft_target(use_cuda_graph: bool, attn_backend: str):
     spec_config = DraftTargetDecodingConfig(
         max_draft_len=max_draft_len,
         speculative_model_dir=draft_model_dir,
+        enable_torch_sampler=True,
```
Yes, we don't need this line. Removing and pushing.

dcampora committed:
Removed unnecessary setting of Torch sampler to false.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented on tensorrt_llm/_torch/pyexecutor/config.py:
```diff
@@ -54,10 +54,9 @@ class PyTorchConfig:
     corresponding sampling strategy, e.g. top-k, top-p, etc.
     """
-    enable_trtllm_sampler: bool = False
+    enable_torch_sampler: bool = False
```
I agree, `use` is better. Changing to that.

dcampora committed:
Change enable_torch_sampler to use_torch_sampler.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

netanel-haber committed:
finalize_events: dict[str, CudaEvent]** | None = None**
`Optional` to accommodate `_forward_step_inter_pp` which creates a `SampleState` without `finalize_events`

Signed-off-by: Netanel Haber <nhaber@nvidia.com>

netanel-haber committed:
wording

Signed-off-by: Netanel Haber <nhaber@nvidia.com>

something

Signed-off-by: Netanel Haber <nhaber@nvidia.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #13310](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13310/) [ run ] triggered by Bot

dcampora committed:
Fix remaining single gpu issues.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

Funatiq commented on tensorrt_llm/llmapi/llm_args.py:
```diff
@@ -1895,11 +1895,11 @@ class TorchLlmArgs(BaseLlmArgs):
         "If true, will iterate over sampling_params of each request and use the corresponding sampling strategy, e.g. top-k, top-p, etc.",
         status="beta")
 
-    enable_trtllm_sampler: bool = Field(
```
Do we need some kind of deprecation warning or hint for the user regarding this change of args?

Funatiq commented on tests/integration/defs/disaggregated/test_configs/disagg_config_trtllm_sampler.yaml:
```diff
@@ -11,7 +11,7 @@ context_servers:
   tensor_parallel_size: 1
   pipeline_parallel_size: 1
-  enable_trtllm_sampler: True
+  use_torch_sampler: False
```
This shouldn't be needed as it is the default case. I'm not sure if that means the entire test case is redundant.

dcampora replied:
Moved it to be "disagg_config_torch_sampler.yaml" instead.

Funatiq commented on tests/unittest/_torch/modeling/test_modeling_nemotron_h.py:
```diff
@@ -41,7 +41,7 @@ def create_nemotron_h_llm(use_cuda_graph, disable_overlap_scheduler,
         disable_overlap_scheduler=disable_overlap_scheduler,
         kv_cache_config=KvCacheConfig(enable_block_reuse=False),
-        enable_trtllm_sampler=True,
+        use_torch_sampler=False,
```
This seems unnecessary as it is the default.

dcampora replied:
Removed.

Funatiq reviewed (commented):
We should mark this as [BREAKING CHANGE] in the PR title. Please also add a Jira task.

dcampora committed:
Fix remaining failing tests.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

tensorrt-cicd commented:
[PR_Github #13405](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13405/) [ run ] triggered by Bot

dcampora committed:
Apply test correction.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run

tensorrt-cicd commented:
[PR_Github #13405](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13405/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #10027](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10027/) completed with status: 'FAILURE'

Funatiq commented:
/bot run

tensorrt-cicd commented:
[PR_Github #13522](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13522/) [ run ] triggered by Bot

netanel-haber commented on tests/integration/test_lists/qa/examples_test_list.txt:
```diff
@@ -603,7 +603,7 @@ disaggregated/test_disaggregated.py::test_disaggregated_deepseek_v3_lite_fp8_att
 disaggregated/test_disaggregated.py::test_disaggregated_load_balance[TinyLlama-1.1B-Chat-v1.0]
 disaggregated/test_disaggregated.py::test_disaggregated_cache_aware_balance[TinyLlama-1.1B-Chat-v1.0]
-disaggregated/test_disaggregated.py::test_disaggregated_trtllm_sampler[TinyLlama-1.1B-Chat-v1.0]
+disaggregated/test_disaggregated.py::test_disaggregated_torch_sampler[TinyLlama-1.1B-Chat-v1.0]
```
Took the liberty, since I was fixing it in my branch as well

netanel-haber commented:
/bot run

tensorrt-cicd commented:
[PR_Github #13886](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13886/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #13886](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/13886/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #10451](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10451/) completed with status: 'FAILURE'

dcampora committed:
test_disaggregated_torch_sampler

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

tensorrt-cicd commented:
[PR_Github #14031](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14031/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #10580](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10580/) completed with status: 'FAILURE'

dcampora committed:
Use torch sampler for tests that use large models.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora committed:
Adapt to use_torch_sampler.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run

dcampora committed:
Fix test_openai_chat_multimodal.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

netanel-haber committed:
setup is_attention_dp_dummy=True gen requests

Signed-off-by: Netanel Haber <nhaber@nvidia.com>

tensorrt-cicd commented:
[PR_Github #14178](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14178/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #10703](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10703/) completed with status: 'FAILURE'

dcampora committed:
Use torch sampler in multimodal.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

netanel-haber committed:
minimal disagg fix

Signed-off-by: Netanel Haber <58652339+netanel-haber@users.noreply.github.com>

tensorrt-cicd commented:
[PR_Github #14274](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14274/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #14262](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14262/) [ run ] completed with state `ABORTED`

syuoni reviewed (approved):
LGTM

DomBrown reviewed (approved):
Based on our discussion on Slack, happy to approve the API change parts.

Shixiaowei02 reviewed (approved):
LGTM for the doc changes. Thanks

dcampora committed:
Adapt keywords.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run --disable-fail-fast

tensorrt-cicd commented:
[PR_Github #14274](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14274/) [ run ] completed with state `ABORTED`
[/LLM/main/L0_MergeRequest_PR pipeline #10778](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10778/) completed with status: 'FAILURE'

dcampora committed:
Update expected output in multimodal test.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

tensorrt-cicd commented:
[PR_Github #14312](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14312/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #14303](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14303/) [ run ] completed with state `ABORTED`

dcampora committed:
Put multimodal tests on torch sampler due to memory consumption.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run

tensorrt-cicd commented:
[PR_Github #14333](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14333/) [ run ] triggered by Bot

dcampora commented:
/bot run

tensorrt-cicd commented:
[PR_Github #14398](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14398/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #14398](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14398/) [ run ] completed with state `FAILURE`
[/LLM/main/L0_MergeRequest_PR pipeline #10883](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10883/) completed with status: 'FAILURE'

dcampora commented:
/bot run

tensorrt-cicd commented:
[PR_Github #14404](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14404/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #14404](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14404/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #10887](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10887/) completed with status: 'FAILURE'

tensorrt-cicd commented:
[PR_Github #14488](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14488/) [ run ] triggered by Bot

dcampora committed:
Waive flaky test_ptp tests.

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>

dcampora commented:
/bot run

tensorrt-cicd commented:
[PR_Github #14512](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14512/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #14512](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/14512/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #10961](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/10961/) completed with status: 'SUCCESS'
